source/common/common/assert.h

```diff
@@ -91,14 +91,45 @@ void invokeDebugAssertionFailureRecordAction_ForAssertMacroUseOnly();
 // _ASSERT_VERBOSE, and this will call _ASSERT_VERBOSE,(__VA_ARGS__)
 #define ASSERT(...)                                                                                \
   EXPAND(_ASSERT_SELECTOR(__VA_ARGS__, _ASSERT_VERBOSE, _ASSERT_ORIGINAL)(__VA_ARGS__))
+
+// When compiled for debug, ASSERT_OR_LOG is simply an ASSERT. This
+// is similar in principle to Chromium's LOG(DFATAL):
+// https://chromium.googlesource.com/chromium/src/+/master/base/logging.h#149
+#define ASSERT_OR_LOG(expr) ASSERT(expr, "ASSERT_OR_LOG()")
```

Comment:
could you add a comment in the description about adding these new macros?

Reply:
done